apps/functions-subscription/src/domain/trial.ts

```diff
@@ -51,6 +51,7 @@ export interface TrialWriter {
 
 export interface TrialReader {
   readonly get: (trialId: TrialId) => TE.TaskEither<Error, O.Option<Trial>>;
+  readonly getAll: () => TE.TaskEither<Error, readonly Trial[]>;
```

Comment:
**suggestion**
```suggestion
  readonly list: () => TE.TaskEither<Error, readonly Trial[]>;
```

I would not use the `getAll` name, because it is highly improbable we will return all the trials